package.json

```diff
@@ -35,10 +35,9 @@
     "@react-navigation/native": "^6.0.1",
     "@react-navigation/stack": "^6.0.1",
     "@reduxjs/toolkit": "^1.6.1",
-    "@types/bson": "^4.0.5",
     "@types/text-encoding": "^0.0.36",
     "big-integer": "^1.6.49",
-    "bson": "^4.4.1",
+    "bson": "4.4.1",
```

Comment:
Why the pinning of the bson version here?

Reply:
Realm has `bson@4.4.1` as a dependency and it was clashing with our unpinned version of `bson` causing a couple of typescript errors.